src/cli/SampleBufferStream.ts

```diff
@@ -0,0 +1,45 @@
+import { Transform } from "stream";
+
+export class SampleBufferStream<
```

Comment:
Extremely don't use this - it will fail when the hop size is greater than the contents of the buffer.